doc/nrf/ug_zigbee_platform_design.rst

```diff
@@ -0,0 +1,99 @@
+.. _ug_zigbee_platform_design:
+
+Zigbee platform designs
+#######################
+
+This page describes the platform designs that are possible with the Zigbee stack on Nordic Semiconductor devices.
+
+The designs are described from the least to the most complex, that is from simple applications that consist of a single chip running single or multiple protocols to scenarios in which the nRF SoC acts as a network co-processor when the application is running on a much more powerful host processor.
+
+The Zigbee stack supports:
+
+* :ref:`ug_zigbee_platform_design_soc`
+* :ref:`ug_zigbee_platform_design_ncp`
+
+.. _ug_zigbee_platform_design_soc:
+
+System-on-Chip designs
+**********************
+
+This single-chip solution has the combined RFIC (the IEEE 802.15.4 in case of Zigbee) and processor. The ZIgbee stack and the application layer run on the local processor.
+
+Single-chip, single protocol (SoC)
+==================================
+
+In this design, the application layer and the stack run on the same processor. The application uses the ZBOSS APIs directly.
+
+This is the SoC design most commonly used for end devices and routers.
+
+This design has the following advantages:
```

Comment:
The pros and cons that are listed here are relevant for both single-phy and multiprocol. I'd consider moving them into ```System-on-Chip designs``` section.

Reply:
I agree. They are common. This changes a bit the scheme, but this should not be an issue.